Sources/TensorFlow/Loss.swift

```diff
@@ -228,6 +239,7 @@ func _vjpSoftmaxCrossEntropyHelper<Scalar: TensorFlowFloatingPoint>(
 }
 
 /// Returns the softmax cross entropy (categorical cross entropy) between logits and labels.
+/// Given the logits and probabilites cross entropy computes `reduction(-logits * log(p))`.
```

Comment:
```suggestion
/// Given the logits and probabilities, the softmax cross entropy loss computes `reduction(-logits * log(p))`.
```

Also, is this calculation correct if the softmax is applied along with the cross entropy loss?

Reply:
Yeah, sorry about that, I believe I fixed it in the last commit